app/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "node ./bin/www",
     "lint": "node_modules/.bin/eslint . --no-fix --ext .js",
     "lint:fix": "node_modules/.bin/eslint . --fix --ext .js",
-    "test": "node_modules/.bin/jest --verbose",
+    "test": "node_modules/.bin/jest --verbose --no-cache",
```

Comment:
Were the tests failing because of the caching? Normally we don't want to be disabling this as it makes Jest go 2x as slow unless we have to.

Reply:
Onigurama is native code and it can fail if tests are cached.

Reply:
Was this what was driving you crazy a couple(?) weeks ago where the tests were resolving differently intermittently? Or was that something else...
If so, great to have a solution.

Reply:
Yes, running the routes test, when it loads up the message parser, that loads up Onigurama, it was generally working immediately after a reinstall and run once.  Run it again, and it fails.  Has to do with native code (?) and it altering things, which I guess if the test is cached then it causes issues with rebuilding itself...  beyond me, but that is one of the recommendations other was maxWorkers, but the no-cache appeared to work.